moon/controller_round1.py

```diff
@@ -181,13 +228,40 @@ def circular_pattern(self):
         start_time = self.sim_time
         wait_for_mapping = False
 
-        self.virtual_bumper = VirtualBumper(timedelta(seconds=5), 0.2) # radius of "stuck" area; a little more as the robot flexes
+        self.virtual_bumper = VirtualBumper(timedelta(seconds=5), 0.2, angle_limit=math.pi/16) # radius of "stuck" area; a little more as the robot flexes
```

Comment:
11deg (math.pi/16) is probably too small value?? If you are supposed to turn in place you probably move faster then 3deg/sec?

Reply:
Generally yes but plenty of times you're wiggling out of a tight place where the turn is actually occuring much more slowly. Also, this is parallel to using 0.1m for linear limit (ie it's a relatively small value).